metadata/metadata_test.go

```diff
@@ -555,6 +555,56 @@ var _ = Describe("Metadata", func() {
 		})
 	})
 
+	When("SetAnnotation is called", func() {
+		It("should add the annotation to the object", func() {
+			key := "foo"
+			value := "bar"
```

Comment:
Other tests declare this as `annotations := map[string]string{"foo": "bar"}`. Could you do something similar. Then you could compare directly the maps. It applies to other tests.